helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -48,8 +48,13 @@ spec:
         containers:
           - volumeMounts: {{- toYaml .Values.head.volumeMounts | nindent 12 }}
             name: ray-head
+            {{- if .Values.head.image }}
+            image: {{ .Values.head.image.repository }}:{{ .Values.head.image.tag }}
+            imagePullPolicy: {{ .Values.head.image.pullPolicy }}
```

Comment:
What will happen if `pullPolicy` is not defined?

Reply:
`ImagePullPolicy` is an optional field for Pod, and the default value is [here](https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting).

Reply:
In this PR, both the `repository` and `tag` must be provided when specifying the `image` section. Regarding the `imagePullPolicy`: if it's left unspecified and the `tag` isn't set to "latest", the default becomes "IfNotPresent". However, if the `tag` is "latest" and `imagePullPolicy` is not set, the default is "Always".